rest-assured-tests/src/test/java/TariffManagementServiceClass.java

```diff
@@ -0,0 +1,39 @@
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.*;
+import static org.junit.Assert.assertEquals;
+
+public class TariffManagementServiceClass {
+
+    static Tariff[] getTariffsList() {
```

Comment:
Why should we use static methods instead of creating instances of this object

Reply:
I can't see any reason of creating instances of this class. It only contains realization of test methods.